packages/mdk4/build.sh

```diff
@@ -0,0 +1,20 @@
+#MDK is a proof-of-concept tool to exploit common IEEE 802.11 protocol weaknesses.
+# MDK4 is a new version of MDK3.
+
+#MDK4 is a Wi-Fi testing tool from E7mer of 360, ASPj of k2wrlz, it uses the osdep library from the aircrack-ng project to inject frames on several operating systems. Many parts of it have been contributed by the great aircrack-ng community: Antragon, moongray, Ace, Zero_Chaos, Hirte, thefkboss, ducttape, telek0miker, Le_Vert, sorbo, Andy Green, bahathir, Dawid Gajownik and Ruslan Nabioullin. THANK YOU!
+
+#MDK4 is licenced under the GPLv2 or later.
+
+TERMUX_PKG_MAINTAINER="KimoCoder & Ezmer "
```

Comment:
Are you Kimocoder or Ezmer? 

If not then set it you yourself or @termux. It should be the person responsible for the package in the termux repositories.

Reply:
used @termux